Cargo.toml

```diff
@@ -26,7 +26,6 @@ harness = false
 [features]
 default = []
 force-inprocess = []
-memfd = []
```

Comment:
Let's leave the feature with a comment saying it's deprecated. This will allow us to put off having to make a semver-major version change.

Reply:
Sort of makes sense, but because it's major is `<1`, semver allows to do anything.
It could be minor or patch version dump too.

However, (I just realize) memfd feature is used in the GHA workflow atm, and that also needs some care.
I'd vote for removing memfd feature completely therefore decreasing unnecessary complexity.

what do you think?

Reply:
That position is reasonable. I'll merge this as-is.